corehq/blobs/migrate.py

```diff
@@ -299,6 +319,34 @@ def migrate(self, filename=None, reset=False, max_retry=2, chunk_size=100):
         return processor.run()
 
 
+class ExportByDomain(Migrator):
+    domain = None
+
+    def by_domain(self, domain):
+        self.domain = domain
+        self.iteration_key = self.iteration_key + '/domain=' + self.domain
+
+    def migrate(self, filename=None, reset=False, max_retry=2, chunk_size=100):
+        if not self.domain:
+            raise MigrationError("Must specify domain")
+
+        doc_migrator = self.doc_migrator_class(self.slug, self.couchdb, filename, self.domain)
+
+        progress_logger = CouchProcessorProgressLogger(self.doc_types)
+
+        document_provider = CouchDocumentProvider(self.iteration_key, self.doc_types)
```

Comment:
Ah thanks.I had started looking at implementing that myself. I don't think case attachments are a thing that is deployed yet, so probably won't need the case provider right?